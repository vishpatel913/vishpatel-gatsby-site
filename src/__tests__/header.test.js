import React from "react";
import renderWithTheme from "../utils/test-renderer";
import Header from "../components/header";

describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme(<Header />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
