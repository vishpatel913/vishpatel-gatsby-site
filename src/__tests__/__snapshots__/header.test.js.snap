// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
.c0 {
  background: #fff;
  border-bottom: #E0E0E0 1px solid;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 0 auto;
  max-width: 1200px;
  padding: 1.5rem 2rem;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
}

.c3 {
  color: #000;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-weight: lighter;
  margin: 0.5rem;
}

.c3:hover {
  color: #512DA8;
}

.c6 {
  color: #000;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-weight: lighter;
  margin: 0.5rem;
}

.c6:hover {
  color: #512DA8;
}

.c4 {
  margin: 0 0.5rem;
  height: 64px;
  width: 64px;
}

@media (max-width:768px) {
  .c1 {
    padding: 1rem;
  }
}

@media (max-width:768px) {
  .c2 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
  }
}

@media (max-width:768px) {
  .c5 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
  }
}

@media (max-width:768px) {
  .c3 {
    font-size: 14px;
  }
}

@media (max-width:768px) {
  .c6 {
    font-size: 14px;
  }
}

@media (max-width:768px) {
  .c4 {
    height: 56px;
    width: 56px;
  }
}

<header
  className="c0"
>
  <div
    className="c1"
  >
    <div
      className="c2"
    >
      <a
        className="c3"
        href="/about"
      >
        ABOUT
      </a>
      <a
        className="c3"
        href="/work"
      >
        WORK
      </a>
    </div>
    <a
      href="/"
      name="Home link"
    >
      <img
        alt="VishPatel.com Logo"
        className="c4"
        src="test-file-stub"
      />
    </a>
    <div
      className="c5"
    >
      <a
        className="c3"
        href="/tech-stack"
      >
        TECH STACK
      </a>
      <a
        className="c6"
        href="test-file-stub"
        rel="noopener noreferrer"
        target="_blank"
      >
        RESUME
      </a>
    </div>
  </div>
</header>
`;
